Makefile

```diff
@@ -107,7 +107,7 @@ $(GOPATH)/bin/gh-release:
 	go get -u github.com/progrium/gh-release/...
 
 $(GOPATH)/bin/go-bindata:
-	go get -u github.com/jteeuwen/go-bindata/...
+	go get -u github.com/minishift/go-bindata/...
```

Comment:
@LalatenduMohanty  Also instead of we fork it into our namespace I would instead use one of maintained fork jteeuwen/go-bindata#3 (comment)

Reply:
This fork seems to be well maintained https://github.com/kevinburke/go-bindata.

Reply:
https://github.com/kevinburke/go-bindata looks better. But I am not sure if we should make the change now. Things are trying to settle down, but I do not think it has yet. personally I would still prefer to use ours fork till the community comes back with the replacement repo i.e. we see PRs , issues and participation in an repository.

Reply:
@LalatenduMohanty, In this case, let's not do anything since `github.com/jteeuwen/go-bindata` is back (done by some other user and agreed to not put any harmful stuff => https://github.com/jteeuwen/go-bindata/issues/5 ). I do agree that we should wait until things get settled. Even we are using our own fork or what currently available under `jteeuwen` name not going to stay for long-term and both contain same stuff till repo disappeared. 

I would suggest let's close this PR and keep eye on Reddit or this newly created repo till everything settled.

Reply:
Similar comment I made here https://github.com/minishift/minishift/issues/1989#issuecomment-364014281